doc/source/examples/incompressible-flow/2d-naca0012-low-reynolds/2d-naca0012-low-reynolds.rst

```diff
@@ -0,0 +1,301 @@
+      set variable             = velocity
+      set fraction type        = number
+      set max number elements  = 700000
+      set max refinement level = 2
+      set min refinement level = 0
+      set frequency            = 5
+      set fraction refinement  = 0.02
+      set fraction coarsening  = 0.16
+    end
+    
+    
+FEM
+~~~
+
+In this example, the interpolation order has been set to one for both velocity and pressure.
+
+.. code-block:: text
+
+    subsection FEM
+      set pressure order = 1
+      set velocity order = 1
+    end
+
+Boundary conditions
+~~~~~~~~~~~~~~~~~~~
+
+The boundary conditions are defined as presented above : 
+
+.. code-block:: text
+
+     subsection boundary conditions
+       set number = 4
+       subsection bc 0
+         set type = noslip
+       end
+       subsection bc 1
+         set type = function
+         subsection u
+           set Function expression = 1
+         end
+         subsection v
+           set Function expression = 0
+         end
+         subsection w
+           set Function expression = 0
+         end
+       end
+       subsection bc 2
+         set type = slip
+       end
+       subsection bc 3
+       	set type = outlet
+       	set beta = 1.3
+       end
+     end
+	
+The boundary 0 corresponds to the NACA, it is a ``noslip`` boundary condition which sets the velocity to zero on the boundary. The boundary 1 is the inlet where the velocity field was chosen to be horizontal and unitary, to ensure that :math:`Re = 1000` is correct, it is represented in green on the image. The boundary 2, in black on the image, is a wall with a ``slip`` boundary condition and finally boundary 3 is of type ``outlet`` with a parameter :math:`\beta = 1.3`. The reader is refered to the `Parameters Guide <https://lethe-cfd.github.io/lethe/parameters/cfd/linear_solver_control.html>`_ for more information about the :math:`\beta` parameter.
+
+Non-linear solver
+~~~~~~~~~~~~~~~~~
+
+The ``inexact_newton`` non-linear solver is used with a high ``tolerance``, since convergence can be hard to obtain for high Reynolds number. The choice of this solver was to reduce the cost of the simulation since it reuses the Jacobian matrix between iterations.
+
+.. code-block:: text
+
+    subsection non-linear solver
+      set solver         = inexact_newton
+      set verbosity      = verbose
+      set tolerance      = 1e-3
+      set max iterations = 10
+    end
+
+Linear solver
+~~~~~~~~~~~~~
+
+Again, in order to reduce the computational time the tolerance for the linear solver were chosen higher than usual : 
+
+.. code-block:: text
+
+    subsection linear solver
+      set verbosity                             = verbose
+      set method                                = gmres
+      set max iters                             = 5000
+      set relative residual                     = 1e-3
+      set minimum residual                      = 1e-8
+      set ilu preconditioner fill               = 1.1
```

Comment:
Why use a fill of 1.1? It's a bit random, I don't think we have ever used non-integer fill level. That's weird :)

Reply:
It is a remnant of me playing with the parameters and should be ignored. Hopefully the results were not impacted by this, I'll suggest using "set ilu preconditioner fill               = 1"